sdk-extensions/incubator/src/test/java/io/opentelemetry/sdk/extension/incubator/fileconfig/OpenTelemetryConfigurationFactoryTest.java

```diff
@@ -74,23 +79,39 @@ class OpenTelemetryConfigurationFactoryTest {
       new DeclarativeConfigContext(
           SpiHelper.create(OpenTelemetryConfigurationFactoryTest.class.getClassLoader()));
 
-  @Test
-  void create_InvalidFileFormat() {
-    List<OpenTelemetryConfigurationModel> testCases =
-        Arrays.asList(
-            new OpenTelemetryConfigurationModel(),
-            new OpenTelemetryConfigurationModel().withFileFormat("1"));
+  @ParameterizedTest
+  @MethodSource("fileFormatArgs")
+  void create_FileFormat(String fileFormat, boolean isValid) {
+    OpenTelemetryConfigurationModel model =
+        new OpenTelemetryConfigurationModel().withFileFormat(fileFormat);
 
-    List<Closeable> closeables = new ArrayList<>();
-    for (OpenTelemetryConfigurationModel testCase : testCases) {
+    if (isValid) {
+      assertThatCode(() -> OpenTelemetryConfigurationFactory.getInstance().create(model, context))
+          .doesNotThrowAnyException();
+    } else {
       assertThatThrownBy(
-              () -> OpenTelemetryConfigurationFactory.getInstance().create(testCase, context))
+              () -> OpenTelemetryConfigurationFactory.getInstance().create(model, context))
           .isInstanceOf(DeclarativeConfigException.class)
-          .hasMessage("Unsupported file format. Supported formats include: 0.4");
-      cleanup.addCloseables(closeables);
+          .hasMessage("Unsupported file format. Supported formats include 0.4, 1.0*");
     }
   }
 
+  private static Stream<Arguments> fileFormatArgs() {
+    return Stream.of(
+        // Invalid file formats
+        Arguments.of(null, false),
+        Arguments.of("0.3", false),
+        Arguments.of("a0.4", false),
+        Arguments.of("0.4a", false),
+        Arguments.of("foo", false),
+        Arguments.of("1.0-rc.a", false),
```

Comment:
Just to make it very clear we're not doing semver....
```suggestion
        Arguments.of("1.0.0", false),
        Arguments.of("1.0.3", false),
        Arguments.of("1.0-rc.a", false),
```